src/tools/annotation/CircleRoiTool.js

```diff
@@ -452,7 +454,8 @@ function _formatArea(area, hasPixelSpacing) {
     ? ` mm${String.fromCharCode(178)}`
     : ` px${String.fromCharCode(178)}`;
 
-  return `A: ${toGermanNumberStringTemp(area)} ${suffix}`; //`Area: ${numbersWithCommas(area.toFixed(2))}${suffix}`;
+  // TODO: localisation!
+  return `A: ${toGermanNumberStringTemp(area)} ${suffix}`; // `Area: ${numbersWithCommas(area.toFixed(2))}${suffix}`;
```

Comment:
Why is `toGermanNumberStringTemp` still being used here, whereas it has been discarded in the other tools?